drupal/modules/avoindata-newsfeed/templates/avoindata_newsfeed_block.html.twig

```diff
@@ -19,35 +19,35 @@
         </h2>
     </div>
 
-    <div class="avoindata-newsfeed-items mt-2">    
+    <div class="avoindata-newsfeed-items mt-2">
 
         {% for news in articles %}
           <div class="avoindata-newsfeed-line h-100 mr-4">
-          </div> 
+          </div>
 
           <div class="avoindata-newsfeed-item mt-2">
             <div class="avoindata-newsfeed-item-date">
                 <p>
                 {% trans %}
-                  Julkaistu 
+                  Julkaistu
```

Comment:
Not really related to this PR, but this should be in english as "Published" and translated in transifex to different languages, now its the other way around